model/src/main/kotlin/config/PathExcludeReason.kt

```diff
@@ -72,12 +72,5 @@ enum class PathExcludeReason {
      * The path only contains tools used for testing source code which are not included in distributed build
      * artifacts.
      */
-    @Deprecated(
```

Comment:
Commit message: Please use a tag instead of a branch for the link:

https://github.com/spdx/spdx-spec/blob/v2.2.2/chapters/relationships-between-SPDX-elements.md